### PR TITLE
Fixdocker

### PR DESCRIPTION
**Pull request by colincoleman**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,9 @@
 FROM grafana/grafana:5.2.2
-RUN curl -L https://github.com/telia-oss/aws-env/releases/download/v0.3.0/aws-env-linux-amd64 > /tmp/aws-env && \
-      echo f80addd4adf9aa8d4ecf1b16de402ba4  /tmp/aws-env | md5sum -c && \
-      chmod +x /tmp/aws-env
+USER root
+RUN curl -L https://github.com/telia-oss/aws-env/releases/download/v0.3.0/aws-env-linux-amd64 > /usr/local/bin/aws-env && \
+      echo f80addd4adf9aa8d4ecf1b16de402ba4  /usr/local/bin/aws-env | md5sum -c && \
```
Comment (mikael-lindstrom) [inline docker/Dockerfile:4]: What is this magic string?

**Review by mikael-lindstrom — APPROVED** [review]
LGTM but it would be nice to clarify my comment 👍

File: docker/Dockerfile (hunk 1 above)
Comment (colincoleman) [reply docker/Dockerfile:4]: This downloads aws-env and checks it against the md5 checksum (that I manually calculated)

Comment (mikael-lindstrom) [reply docker/Dockerfile:4]: Ahh, makes sense 👍

**Pull request merged**